contributed_definitions/nyaml/NXraman.yaml

```diff
@@ -0,0 +1,755 @@
+            defined incident or scattered light state.
+          enumeration: [Polarization by Fresnel reflection, Birefringent polarizers, Thin film polarizers, Wire-grid polarizers, other]
+        specific_polarization_filter_type(NX_CHAR):
+          doc: |
+            Specific name or type of the polarizer used.
+            
+            Free text, for example: Glan-Thompson, Glan-Taylor, Rochon Prism, Wollaston
+            Polarizer...
+      spectral_filter_N(NXbeam_device):
+        filter_type:
+          exists: optional
+          doc: |
+            Type of laserline filter used to supress the laser, if measurements
+            close to the laserline are performed.
+          enumeration: [long-pass filter, short-pass filter, Notch filter, reflection filter, neutral density filter, other]
+        intended_use:
+          exists: optional
+          doc: |
+            Type of laserline filter used to supress the laser, if measurements
+            close to the laserline are performed.
+          enumeration: [laser line cleanup, raylight line removal, spectral filtering, intensity manipulation, other]
+        filter_characteristics(NX_NUMBER):
+          exists: optional
+          doc: |
+            Properties of the spectral filter such as wavelength dependent Transmission
+            or reflectivity.
+          dimensions:
+            rank: 2
+            dim: [[1, 2], [2, N_spectrum_filter]]
+            # 2 -> [Wavelength, Transmission]
+            # N_spectrum_filter --> Number of wavelengths
+          \@characteristics_type:
+            exists: optional
+            doc: |
+              Which property is used to form the spectral properties of light,
+              i.e. transmission or reflection properties.
+            enumeration: [transmission, reflection]
+      opt_monochromator(NXbeam_device):
+        exists: optional
+        doc: |
+          Dispersive element to create e.g. an energetic resolution used in the 
+          measurement.
+        device:
+          doc: |
+            It should contain the name of the Nexus Monochromator group
+      (NXmonochromator):
+        doc: |
+          Monochromator with arbitrary choice of the name, which shall be referenced
+          by the spectrometer(NXbeam_device).
+      Sample_as_beam_element(NXbeam_device):
+        doc: |
+          Enables beam path description of the sample in the setup.
+          it in a beam path.
+        device:
+          doc: |
+            Contains the link to the sample.
+    (NXsample):
+    data_collection(NXprocess):
+      data_type:
+        doc: |
+          Select which type of data was recorded, for example a simple spectrum
+          with intensity vs. Raman shift, CCD image or a set of spectra with
+          probe, reference and background signals for pump beam on and off.
+        enumeration: [Integrated single spectrum, CCD image, Probe + Background + Reference for Pump-on and Pump-off (i.e. for FSRS)]
```

Comment:
I don't understand this part. `NXprocess` is intended to describe post-processing of data. Here it's just used to denote which typo of data was collected. For me this rather belongs into the (NXdetector) group or so

Reply:
This is defined in this way in NXopt. Maybe thats a remaining thing from the Harmonization of NXellipsometry and NXopt?
Right, from the definition of NXprocess, this does not seem to be proper.
No action done yet - have to discuss this.